scipy/stats/morestats.py

```diff
@@ -2441,6 +2441,7 @@ def levene(*args, center='median', proportiontocut=0.05):
     return LeveneResult(W, pval)
 
 
+@np.deprecate(new_name='binomtest')
```

Comment:
should the message indicate the SciPy version in which `binom_test` goes away entirely?

IIRC Under our policy it would be 2 release cycles, so version 1.8.x

Reply:
I used the `message` parameter to add information about when the function will be removed.